backward.hpp

```diff
@@ -1391,23 +1391,22 @@ class TraceResolverLinuxImpl<trace_resolver_tag::libdw>:
 					if (die_has_pc(die, pc)) {
 						return result;
 					}
-				default:
```

Comment:
As mentioned in the commit message, there is an implicit fallthrough
(a fallthrough is a missing break after a case in a switch). There are
compiler-based and C++17-based ways to explicitly mark fallthroughs
but the easiest way to get rid off the warning is to move the code out,
isn't it?